Options.py

```diff
@@ -843,11 +844,11 @@ class OptionList(Option[typing.List[typing.Any]], VerifyKeys):
     # If only unique entries are needed and input order of elements does not matter, OptionSet should be used instead.
     # Not a docstring so it doesn't get grabbed by the options system.
 
-    default: typing.List[typing.Any] = []
+    default: typing.Union[typing.List[typing.Any], typing.Tuple[typing.Any, ...]] = ()
     supports_weighting = False
 
-    def __init__(self, value: typing.List[typing.Any]):
-        self.value = deepcopy(value)
+    def __init__(self, value: typing.Iterable[str]):
+        self.value = list(deepcopy(value))
```

Comment:
I should have noticed this. You changed the type var from `Any` to `str`, and I trusted it.

Reply:
I missed it as well. We can at least partially blame the lack of unit tests :-)

Reply:
Well, I should have checked deeper how this was used. Definitely my bad on this one.